diam/dict/default.go

```diff
@@ -13,11 +13,28 @@ import "bytes"
 var Default *Parser
 
 func init() {
-	Default, _ = NewParser()
-	Default.Load(bytes.NewReader([]byte(baseXML)))
-	Default.Load(bytes.NewReader([]byte(creditcontrolXML)))
-	Default.Load(bytes.NewReader([]byte(networkaccessserverXML)))
-	Default.Load(bytes.NewReader([]byte(tgpprorfXML)))
+	var err error
+
+	Default, err = NewParser()
+	if err != nil {
+		panic(err)
+	}
```

Comment:
Don't really think this is needed but if you want to go this route let's please loop over an array